src/Persistance/RepositoryImplementations/SystemRegisterRepository.cs

```diff
@@ -181,13 +171,59 @@ UPDATE altinn_authentication.system_register
         }
     }
 
+    /// <inheritdoc/> 
+    public async Task<List<DefaultRights>> GetDefaultRightsForRegisteredSystem(Guid systemId)
+    {
+        const string QUERY = /*strpsql*/@"
+                SELECT unnest default_rights
+                FROM altinn_authentication.system_register
+                WHERE altinn_authentication.system_register.registered_system_id = @registered_system_id;
+                ";    
+
+        try
+        {
+            await using NpgsqlCommand command = _datasource.CreateCommand(QUERY);
+
+            command.Parameters.AddWithValue("registered_system_id", systemId);
+
+            return await command.ExecuteEnumerableAsync()
+                .SelectAwait(ConvertFromReaderToDefaultRights)
+                .ToListAsync();
+                         
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Authentication // SystemRegisterRepository // GetDefaultRightsForRegisteredSystem // Exception");
+            throw;
+        }
+    }
+
+    /// <summary>
+    /// The list of DefaultRight for each Registered System is stored as a text array in the db.
+    /// Each element in this Array Type is a concatenation of the Servive Provider ( NAV, Skatteetaten, etc ...)
+    /// and the Right joined with an underscore.
+    /// This is to avoid a two dimensional array in the db, this is safe and easier since
+    /// each Right is always in the context of it's parent Service Provider anyway.
+    /// The Right can either denote a single Right or a package of Rights; which is handled in Access Management.
```

Comment:
I hope the return types are agreed with the tilgangstyring team

Reply:
We had a small discussion in the office, and agreed to have a meeting to discuss this in detail. So this will probably be refactored anyway in a later PR. Suggest closing this comment for this PR.